lib/python/qmk/cli/list/keymaps.py

```diff
@@ -13,5 +13,9 @@
 def list_keymaps(cli):
     """List the keymaps for a specific keyboard
     """
-    for name in qmk.keymap.list_keymaps(cli.config.list_keymaps.keyboard):
-        print(name)
+    if cli.config.list_keymaps.keyboard:
+        for name in qmk.keymap.list_keymaps(cli.config.list_keymaps.keyboard):
+            print(name)
+    else:
+        cli.log.error('Could not determine keyboard!')
+        return False
```

Comment:
```suggestion
    if not cli.config.list_keymaps.keyboard:
        cli.log.error('Missing required arguments: --keyboard')
        cli.subcommands['list-keymaps'].print_help()
        return False

    for name in qmk.keymap.list_keymaps(cli.config.list_keymaps.keyboard):
        print(name)
```

Reply:
Was referencing `compile`, but yeah, this is probably better.